game-core/build.gradle

```diff
@@ -134,7 +133,6 @@ dependencies {
         testCompile 'org.testfx:openjfx-monocle:8u76-b04'
     }
 
-    testCompileOnly 'org.apiguardian:apiguardian-api:1.0.0'
```

Comment:
I couldn't reproduce #2524 most likely due to the usage of "implementation" instead of "compile" haven't bothered to confirm this theory though